README.md

```diff
@@ -50,21 +50,22 @@ val sqlContext = new SQLContext(sc)
 // Get some data from a Redshift table
 val df: DataFrame = sqlContext.read
     .format("com.databricks.spark.redshift")
-    .option("url", "jdbc:redshift://redshifthost:5439/database?user=username&password=pass")
-    .option("dbtable" -> "my_table")
-    .option("tempdir" -> "s3://path/for/temp/data")
+    .option("url", "jdbc:postgresql://redshifthost:5439/database?user=username&password=pass")
```

Comment:
This should be `redshift`. In an earlier commit, I updated the guide to illustrate use of Amazon's official JDBC connector.